recipes/binpacking/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "binpacking" %}
+{% set version = "1.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f2bd30b341406708305ae29e51959f486bfd3fe68f5b7d1a963489a6353f0cbc
+
+build:
+  number: 0
+  entry_points:
+    - binpacking = binpacking.binpacking_binary:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir"
+
+requirements:
+  host:
+    - pip
+    - python
+    - numpy
+    - future
+    - pytest
+    - pytest-cov
+    - pytest-runner
+  run:
+    - pip
```

Comment:
I guess `pip` isn't a runtime dependency?
```suggestion
```

Reply:
you're right, done